test/test_custom_types.py

```diff
@@ -255,7 +255,7 @@ def fallback_encoder(value):
 
 class TestBSONTypeEnDeCodecs(unittest.TestCase):
     def test_instantiation(self):
-        msg = "Can't instantiate abstract class .* with abstract methods .*"
+        msg = "Can't instantiate abstract class"
```

Comment:
Instead of outright removing the latter half of this regex, might it be acceptable to use the following instead? 
`Can't instantiate abstract class .* with abstract method.* .*`

Reply:
Isn't `Can't instantiate abstract class` enough to ensure the TypeError is from instantiation?

Reply:
👍